elements/pl-checkbox/pl-checkbox.css

```diff
@@ -2,3 +2,7 @@
     width: 340px;
     max-width: 100%;
 }
+
+:not(.form-check-inline) > .pl-checkbox-key-label {
+    min-width: 20px;
```

Comment:
Not that it will really change anything, but would it be better to use `em` as the units here, given that this is sizing for text?

Reply:
Good point!